moving-forward-series/simple-payment-verification.md

```diff
@@ -0,0 +1,31 @@
+# What is Simple Payment Verification (SPV)  - Moving forward (better title needed)
+
+There is a need for mobile wallets that are fast to set up, easy to use and very secure. All too often, mobile wallets sacrifice the security of a fully validating wallet for easy of use. SPV challenges this convention by providing a high level of security whilst improving the user experience. 
+
+## Fast and Small
+If you’re looking for a non-custodial mobile wallet, Simple Payment Verification (SPV) is an idea solution.  SPV is designed to be intentionally lightweight, as it interacts with the blockchain by only fetching and downloads the data, that is absolutely necessary. This includes block headers, block filters, and the full blocks that contain transaction data, directly associated with the wallet. As you can imagine, this reduces the size, set up and sync times considerably, from hours to minutes. In Decred’s case, the setup of a new fully validating node is approximately 12 GB and its SPV counterpart is approximately 0.6 GB.
```

Comment:
Let's have a blank line after each heading. It should look the same when rendered but it reads better in text editors and monospace fonts.

Personally I use *two* blank lines before a heading and one after to get even better UX in monospace text editors, but it's up to you.